src/pages/workspace/WorkspacesListRow.tsx

```diff
@@ -142,9 +142,9 @@ function WorkspacesListRow({
     const isDeleted = style && Array.isArray(style) ? style.includes(styles.offlineFeedback.deleted) : false;
 
     const ThreeDotMenuOrPendingIcon = (
-        <View style={[isNarrow && styles.mr5]}>
+        <View style={[styles.flexRow]}>
```

Comment:
This is not really the culprit PR of https://github.com/Expensify/App/issues/44639.
However if we don't add limit width here and with the `flex1` style of `Badge` wrapper View, it will cause https://github.com/Expensify/App/issues/44639.
More detail in this proposal RCA:
https://github.com/Expensify/App/issues/44639#issuecomment-2197991216